ascii.py

```diff
@@ -0,0 +1,31 @@
+class Ascii():
+
+	def __init__(self):
+		pass
+
+	def draw_square(self, piece):
+		space = "| " + piece + " "
+		return space
+
+	def draw_row(self, row_marker, pieces):
+		bottom_half = "|\n    " + "|____" * 8 + "|"
+		row = " " + row_marker + "  "
+		for piece in pieces:
+			row += (self.draw_square(piece))
+		row += bottom_half
+		print(row)
+
+	def draw_board(self, board):
+		board.get_map()
+		top_row = "     " + "_" * 39
```

Comment:
39 is a magic number here, can we give it either a constant name or an expression that explains where it derives from? I assume the logic here is something like:

```
CELL_WIDTH = 4
BORDER_WIDTH = 1
WIDTH = CELL_WIDTH * 8 + BORDER_WIDTH * (8-1)
```

Reply:
I've never heard the term "magic number" in this context but it feels like a buzzword. I def understand what you mean in context. I'll replace it with a well-named variable and probably comment it so it doesn't feel so out of nowhere.